region/brinstar/blue/Blue Brinstar Boulder Room.json

```diff
@@ -152,6 +153,7 @@
           "framesRemaining": 60
         }
       },
+      "unlocksDoors": [{"types": ["ammo"], "requires": []}],
```

Comment:
Good point, added a "canTrickyJump" req in those cases now. It really does seem pretty tricky because you need to know basically exactly where the end of the invisible ledge is.